packages/tools/viewer-alpha/src/viewer.ts

```diff
@@ -190,6 +191,11 @@ export class Viewer implements IDisposable {
      */
     public readonly onModelError = new Observable<unknown>();
 
+    /**
+     * Fired when progress changes on loading activity.
+     */
+    public readonly onLoadingProgressChanged = new Observable<void>();

```

Comment:
wouldn't it be better for developers if the observable notified the actual progress and not just void?

Reply:
Talked offline, will leave it as is for now.